### PR TITLE
Vaida Week 3

### PR DESCRIPTION
**Pull request by Ereja**

### CODE REVIEW

File: Week3/homework/3-injection.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    throw err;
+  }
+  console.log(result);
+});
+
+//Rewrite the function so that it is no longer vulnerable to SQL injection
+function getPopulationFixed(Country, name, code, cb) {
+  //adding placeholders
+  conn.query(
+    `SELECT Population FROM ${Country} WHERE Name = ? and code = ?`,
```
Comment (GinaSt) [inline Week3/homework/3-injection.js:26]: You can also use a ?? placeholder for identifiers.

File: Week3/homework/2-transactions/transactions-create-tables.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  const execQuery = util.promisify(connection.query.bind(connection));
+
+  try {
+    await execQuery(accountTable);
+    await execQuery(accountChangesTable);
+    connection.end();
+    process.exit(0);
+  } catch (err) {
+    console.error(err.message);
+    connection.end();
```
Comment (GinaSt) [inline Week3/homework/2-transactions/transactions-create-tables.js:40]: You could put a single _connection.end();_  right after the _try...catch_  or you could add a _try-catch-finally_ statement and close your connection in the _finally_ statement

**Review by GinaSt — APPROVED** [review]
Nice Work!
I think your homework is complete - you seem to have understood all the concepts. Please have a look at my comments. Maybe you will get some more ideas for improvements.

File: Week3/homework/1-normalization.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+ANSWER:
+members table:
+member_id INT PRIMARY KEY AUTO_INCREMENT,
+member_name VARCHAR(50) NOT NULL,
+member_address VARCHAR(255) NOT NULL
+
+dinner table:
+dinner_id INT PRIMARY KEY AUTO_INCREMENT,
+dinner_date DATE NOT NULL,
+member_id INT
```
[the remark this answers is not in the document]
Comment (Ereja) [reply Week3/homework/1-normalization.md:30]: Very good point! Somehow the order relationship seemed so obvious, but this one slipped through my mind :)

File: Week3/homework/3-injection.js (hunk 1 above)
Comment (Ereja) [reply Week3/homework/3-injection.js:26]: Thank you, that`s something I didn`t knew before

**Comment by Ereja** [comment]
@GinaSt thank you for such nice feedback, I have learned a few new things from it! :)

**Pull request closed without merging**